$NetBSD: patch-man_man5_configure.acr.5,v 1.2 2013/06/12 11:00:57 ryoon Exp $

* WIth this patch, configure.acr(5) is rendered properly with mdocml.

--- man/man5/configure.acr.5.orig	2011-08-04 10:24:58.000000000 +0000
+++ man/man5/configure.acr.5
@@ -33,14 +33,13 @@ generation of .acr files with SUBDIRS an
 .El
 .\"
 .Sh SYNTAX
-.Bl -tag -width indent
 The configure.acr file is a word-based configuration file for ACR.
 .Pp
 This means that each word must be separated with space, tab or new-lines.
 The ';' separator to terminate strings must be also a separated word.
 .Pp
-.Bl -bold -tag -width indent
 Each word is called "keyword", there are these types:
+.Bl -bold -tag -width indent
 .It Miscelaneous-Keys:
 { } ;
 .It Command:
@@ -208,8 +207,8 @@ Checks the existence of the target user 
 .El
 .\"
 .Sh DEFINITIONS
-.Bl -tag -width indent
 Sets are processed at the beggining of the configure script.
+.Bl -tag -width indent
 .It = varname value\ ;
 Sets the value of varname to value.
 .It += varname value\ ;
